include/zephyr/bluetooth/gatt.h

```diff
@@ -1657,15 +1657,30 @@ struct bt_gatt_discover_params {
 		struct {
 			/** Include service attribute declaration handle */
 			uint16_t attr_handle;
-			/** Included service start handle */
+			/** Starting attribute handle for included service */
 			uint16_t start_handle;
-			/** Included service end handle */
+			/** Ending attribute handle for included service */
 			uint16_t end_handle;
 		} _included;
-		/** Discover start handle */
+		/** Starting attribute handle to begin discovery
+		 *
+		 * @note When discovery begins in @ref bt_conn_cb.connected
+		 * callback this is typically set to
```

Comment:
The `bt_conn_cb.connected` callback is irrelevant w.r.t. this function. It works the same regardless of what function it's called in. Suggest to omit reference to `bt_conn_cb.connected`